sdk/aqueduct/artifacts/preview.py

```diff
@@ -50,13 +57,6 @@ def preview_artifacts(
         ],
         make_copy=True,
     )
-
-    global_engine_config: Optional[EngineConfig] = None
-    if globals.__GLOBAL_CONFIG__.engine is not None:
-        global_engine_config = generate_engine_config(
-            globals.__GLOBAL_API_CLIENT__.list_integrations(),
-            globals.__GLOBAL_CONFIG__.engine,
-        )
```

Comment:
non-functional change. It just looks better to have the two subgraph statements next to each other.